packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -241,6 +241,15 @@ Future<DateTime?> showDatePicker({
       locale: locale,
       child: dialog,
     );
+  }else{
```

Comment:
```suggestion
  } else {
```
nit: spacing

Reply:
Thanks for your suggestion, I've commit it
@MitchellGoodwin